internal/fsutil/create.go

```diff
@@ -15,6 +15,8 @@ type CreateOptions struct {
 	Path string
 	Mode fs.FileMode
 	Data io.Reader
+	// If Link is set and the symlink flag is set in Mode, a symlink is
+	// created. Otherwise, a hard link is created.
```

Comment:
I think this is a bit confusing because it is unclear that the "otherwise" applies only to the second part of the sentence. I realize my previous suggestion has the same problem so let's change it to be more explicit. What about something like?
```
	// If Link is set and the symlink flag is set in Mode, a symlink is
	// created. If the Mode is not set to symlink, a hard link is created
	// instead.
```

Reply:
Fixed in [3e8d872](https://github.com/letFunny/chisel/pull/17/commits/3e8d872c21ef3ed81e75c5319a052f872feb8b25).